src/test/java/org/terracotta/statistics/derived/histogram/BarSplittingBiasedHistogramFittingTest.java

```diff
@@ -0,0 +1,153 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.derived.histogram;
+
+import org.apache.commons.math3.fitting.AbstractCurveFitter;
+import org.apache.commons.math3.fitting.GaussianCurveFitter;
+import org.apache.commons.math3.fitting.PolynomialCurveFitter;
+import org.apache.commons.math3.fitting.WeightedObservedPoint;
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+import static org.hamcrest.number.IsCloseTo.closeTo;
+import static org.junit.Assert.assertThat;
+
+@RunWith(Parameterized.class)
+public class BarSplittingBiasedHistogramFittingTest {
+
+  private static final double ERROR_THRESHOLD = 10;
+
+  private final long seed;
+  private final float bias;
+  private final int bars;
+
+  private final double slopeError;
+  private final double widthError;
+  private final double centroidError;
+
+  @Parameterized.Parameters(name = "{index}: seed={0}, bias={1}, bars={2}")
+  public static Iterable<Object[]> data() {
+    Random rndm = new Random();
+    // seed, bias, bars, slope-error, centroid-error, width-error
+    return Arrays.asList(new Object[][] {
+            {rndm.nextLong(), 0.01f, 20, 0.00205, 0.0153, 0.0121},
+            {rndm.nextLong(), 0.01f, 100, 0.00202, 0.00551, 0.00479},
+            {rndm.nextLong(), 0.01f, 1000, 0.00248, 0.00509, 0.00497},
+
+            {rndm.nextLong(), 0.1f, 20, 0.00137, 0.00710, 0.00746},
+            {rndm.nextLong(), 0.1f, 100, 0.00130, 0.00511, 0.00510},
+            {rndm.nextLong(), 0.1f, 1000, 0.00134, 0.00480, 0.00509},
+
+            {rndm.nextLong(), 1f, 20, 0.00118, 0.00557, 0.00703},
+            {rndm.nextLong(), 1f, 100, 0.00110, 0.00508, 0.00532},
+            {rndm.nextLong(), 1f, 1000, 0.00102, 0.00497, 0.00531},
+
+            {rndm.nextLong(), 10f, 20, 0.00134, 0.00712, 0.00797},
+            {rndm.nextLong(), 10f, 100, 0.00124, 0.00499, 0.00500},
+            {rndm.nextLong(), 10f, 1000, 0.00131, 0.00499, 0.00512},
+
+            {rndm.nextLong(), 100f, 20, 0.00193, 0.0151, 0.0123},
+            {rndm.nextLong(), 100f, 100, 0.00199, 0.00527, 0.00471},
+            {rndm.nextLong(), 100f, 1000, 0.00253, 0.00506, 0.00473},
+    });
+  }
+
+  public BarSplittingBiasedHistogramFittingTest(long seed, float biasRange, int bars, double slopeError, double centroidError, double widthError) {
+    this.seed = seed;
+    this.bias = (float) Math.pow(biasRange, 1.0 / bars);
+    this.bars = bars;
+
+    this.slopeError = slopeError;
+    this.centroidError = centroidError;
+    this.widthError = widthError;
+  }
+
+  @Test
+  @Ignore
```

Comment:
Why ignored? (please comment)

Reply:
It's used to establish what the S.D. is on the fit - those values get fed back in to the parameters for the test.

Reply:
Got it. Can you please comment in the code about how it should be used?